app/Http/Controllers/Store/BundleController.php

```diff
@@ -162,10 +181,47 @@ function ($value) {
             $errors[] = $bundle->syncVouchers($voucherCodes);
         }
 
-        // TODO : disbursing bundle code to go here.
+        // Check we have values on our inputs; This should have been covered in validation...
```

Comment:
> This should have been covered in validation...

:)

Reply:
yeah. It should be, but i also thought i'd check they were actually filled here.